linux_os/guide/system/software/updating/ensure_gpgcheck_local_packages/rule.yml

```diff
@@ -46,20 +46,24 @@ references:
     stigid@rhel8: RHEL-08-010371
     vmmsrg: SRG-OS-000366-VMM-001430,SRG-OS-000370-VMM-001460,SRG-OS-000404-VMM-001650
 
-ocil_clause: 'gpgcheck is not enabled or configured correctly to verify local packages'
+ocil_clause: 'there is no process to validate certificates for local packages that is approved by the organization'
 
 ocil: |-
-    To verify that <tt>localpkg_gpgcheck</tt> is configured properly, run the following
-    command:
+    Verify that dnf verifies the signature of local packages prior to install with the following command:
```

Comment:
I think DNF can be replaced by the Jinja variable here.

Reply:
yes